.github/ISSUE_TEMPLATE/001-bug_report.yml

```diff
@@ -0,0 +1,75 @@
+name: Bug report
+description: Report an issue with Kadena
+labels:
+  - bug
+body:
+  - type: markdown
+    attributes:
+      value:
+        Thank you for taking the time to file a bug report. Please fill out this
+        form as completely as possible.
+
+  - type: checkboxes
+    id: initial-checks
+    attributes:
+      label: 'Please confirm the following:'
+      options:
+        - label:
+            I have read the [Code of
+            Conduct](https://github.com/kadena-community/kadena.js/blob/master/CODE_OF_CONDUCT.md)
+          required: true
+        - label: The bug was not reported before
+          required: true
+        - label: I am using the **latest version** of the relevant package(s)
+          required: true
+
+  - type: input
+    id: os
+    attributes:
+      label: What operating system are you using? And what version?
+      placeholder: Mac, Windows, Linux
+    validations:
+      required: false
+
+  - type: input
+    id: browser
+    attributes:
+      label: What environment or browser are you using? And what version?
+      placeholder: Node.js, Chrome, Firefox, Safari
+    validations:
+      required: false
+
+  - type: textarea
+    id: description
+    attributes:
+      label: Describe the bug
+      description: A clear and concise description of the bug
+    validations:
+      required: true
+
+  - type: input
```

Comment:
Probably it should be a textarea if we expect the steps, also in order to have a possibility to add attachments for screenshots/videos/gifs?

Reply:
Oh that was too much copy-pasting from my end. Fixed a few more instances, thanks!